### DISCUSSION TITLE
GooglePay option not shown in payment bottom sheet

### DISCUSSION THREAD

**Discussion by rddewan** [flutter-stripe/flutter_stripe#1003] [Q&A]
I am trying to setup a google_pay put it is never shown in bottom sheet
```
await  Stripe.instance.initPaymentSheet(
          paymentSheetParameters: SetupPaymentSheetParameters(
              paymentIntentClientSecret: paymentIntent['client_secret'],              
              style: ThemeMode.dark,
              merchantDisplayName: 'Richard',
              googlePay: const PaymentSheetGooglePay(
                merchantCountryCode: 'TH',
                currencyCode: 'THB',
                testEnv: true,
              )
            ),
          );
```

**Comment by TheoBo** [comment]
i did have the same problem and could solve it by adding:


`<meta-data
                android:name="com.google.android.gms.wallet.api.enabled"
                android:value="true" />`

to the application section of the AndroidManifest.xml

**Reply by rddewan** [reply]
poor documentation with brilliant product is to just kill the developers time

**Comment by rddewan** [comment]
iOS apple pay is showing but in android I am just getting the bottom sheet with card payment
